docs/adr/adr-0018-how-to-give-feedback-after-a-user-action.md

```diff
@@ -0,0 +1,414 @@
+| JAWS + Chrome     | ✅ Fully supported        | ✅ Fully supported          | ❌ Not supported           | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| JAWS + EDGE       | ✅ Fully supported        | ✅ Fully supported          | ❌ Not supported           | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| JAWS + Firefox    | ❌ Not supported          | ✅ Fully supported          | ⚠ Partially supported (1) | ✅ Fully supported         | ✅ Fully supported                        | ❌ Not supported                  | ✅ Fully supported                       |
+| VO + Safari (Mac) | ✅ Fully supported        | ✅ Fully supported          | ✅ Fully supported         | ⚠ Partially supported (2) | ⚠ Partially supported (2)                | ✅ Fully supported                | ✅ Fully supported                       |
+| VO + Safari (iOS) | ✅ Fully supported        | ✅ Fully supported          | ✅ Fully supported         | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| Talkback + Chrome | ✅ Fully supported        | ✅ Fully supported          | ✅ Fully supported         | ✅ Fully supported         | ✅ Fully supported                        | ⚠ Partially supported (3)        | ✅ Fully supported                       |
+
+**Note** :
+* ✅ Fully supported means the aria-live and aria-atomic settings are respected. However, most of the time, **role semantics are not announced**. We consider this a very minor issue in our case because we already convey this semantic through an icon with an alternative text.
+* ⚠ Partially supported means that we found issues that makes this solution not robust enough for our use case. Usually, something is announced, but it does not respect the aria-atomic setting.
+* ❌ Not supported means nothing is announced.
+
+(1) sometimes not announced at all
+
+(2) wrong announcement
+
+(3) atomicity is not applied
+
+### Comparing our implementation with other libraries
+
+It's always interesting to check differences with other libs.
+We took two different toasters:
+
+* https://fkhadra.github.io/react-toastify/
+  * this one works only with React
+  * There are a lot of features that we don't need
+  * It is 6.5 kb code (https://bundlephobia.com/package/react-toastify@9.0.6)
+* https://github.com/Maronato/vue-toastification
+  * This one works only with Vue.js
+  * It is 6,1 kB of code https://bundlephobia.com/package/vue-toastification@1.7.14
+
+* Both of them don't have what we do for accessibility
```

Comment:
Both of them => Neither of them

Reply:
I think this sentence should not be a list item but be after the two items.